README.md

```diff
@@ -96,7 +96,24 @@ An interface defining the configuration attributes to bootstrap `localStorageSyn
 * `checkStorageAvailability` \(*boolean? = false*): Specify if the storage availability checking is expected, i.e. for server side rendering / Universal.
 * `mergeReducer` (optional) `(state: any, rehydratedState: any, action: any) => any`: Defines the reducer to use to merge the rehydrated state from storage with the state from the ngrx store. If unspecified, defaults to performing a full deepmerge on an `INIT_ACTION` or an `UPDATE_ACTION`.
 
-Usage: `localStorageSync({keys: ['todos', 'visibilityFilter'], storageKeySerializer: (key) => 'cool_' + key, ... })`. In this example `Storage` will use keys `cool_todos` and `cool_visibilityFilter` keys to store `todos` and `visibilityFilter` slices of state). The key itself is used by default - `(key) => key`.
+### Usage
+#### Key Prefix
+```
```

Comment:
Thanks for cleaning this up! Can you add the syntax here and in the below code block please.
```suggestion
```ts
```

Reply:
That's pretty cool, didn't know about that markdown feature!

Reply:
Yeah it's really nice. I suppose it might not work in every markdown editor, but most support it :)